src/System.Net.Http/src/netcore50/project.json

```diff
@@ -16,6 +16,7 @@
         "System.Resources.ResourceManager": "4.0.0",
         "System.Runtime.Extensions": "4.0.10",
         "System.Runtime.WindowsRuntime": "4.0.10",
+        "System.Security.Cryptography.X509Certificates": "4.1.0-rc3-24008-00",
```

Comment:
The point was to make 4.1 stable and ship for RTM.

@stephentoub How do you want to proceed? We cannot update the contract to 4.1 and ship (mark stable) for RTM unless we get Crypto to support UWP for netstandard1.3.